force-app/main/default/triggers/ContactTrigger.trigger

```diff
@@ -16,10 +16,18 @@
  * 
  * Optional Challenge: Use a trigger handler class to implement the trigger logic.
  */
-trigger ContactTrigger on Contact(before insert) {
+trigger ContactTrigger on Contact(before insert, before update) {
```

Comment:
Great job implementing a framework to move the logic out of the trigger itself! Just update the context used for the callouts